docs/RELEASE-NOTES.rst

```diff
@@ -18,6 +18,7 @@ Bug Fixes
 `````````
 * If member already exists, it causes HTTP 409 ERROR
 * Fix requirements
```

Comment:
Looks like the line of "Fix requirements
" is not required, please check if it should be deleted.

Reply:
A requirment fix, noted.